docs/spec/index.md

```diff
@@ -0,0 +1,18 @@
+<!--[metadata]>
++++
+title = "Reference"
+description = "Explains registry JSON objects"
+keywords = ["registry, service, images, repository,  json"]
+[menu.main]
+identifier="smn_registry_ref"
+parent="smn_registry"
+weight=7
++++
+<![end-metadata]-->
+
+# Docker Registry Reference
+
+* [HTTP API V2](api.md)
+* [Storage Driver](../storagedrivers.md)
+* [Token Authentication Specification](auth/token.md)
```

Comment:
I think this is the correct order (specification, then implementation) but it looks like it's alphabetically sorted in the mockup